### PR TITLE
fixes cows moving while tipped

### LINKED ISSUE #36923: cows move when tipped

### PR DESCRIPTION
**Pull request by Iamgoofball**
62fb5f1024529266c6e71c0c0c9ddb3c_xl

Fixes #36923

### CODE REVIEW

**Review by Tlaltecuhtli — APPROVED** [review]

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -177,6 +179,9 @@
 	else
 		..()
 
+/mob/living/simple_animal/cow/proc/reset_canmove()
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: why did you make a proc that does nothing but call another proc am I being trolled

Comment (Cyberboss) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: yes

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 2]
```diff
@@ -157,6 +157,8 @@
 			"<span class='notice'>You tip over [src].</span>")
 		to_chat(src, "<span class='userdanger'>You are tipped over by [M]!</span>")
 		Knockdown(60)
+		update_canmove()
+		addtimer(CALLBACK(src, .proc/reset_canmove), 61)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]:
```dm
addtimer(CALLBACK(src, .proc/update_canmove), 61)
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Jalleo — APPROVED** [review]

**Comment by AnturK** [comment]
Status effects already handle updating canmove why is this necessary and does it actually fix anything ?

**Comment by Iamgoofball** [comment]
i dont think its updating properly on cows

**Comment by Jalleo** [comment]
Then is that due to somewhere else being broken?

**Comment by Iamgoofball** [comment]
thats the thing, cows dont override the movement stuff

after this patch I couldn't get cows moving while pushed to occur so this must fix it

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**